nemo/collections/tts/data/dataset.py

```diff
@@ -453,6 +456,13 @@ def add_pitch(self, **kwargs):
             with open(Path(pitch_stats_path), 'r', encoding="utf-8") as pitch_f:
                 self.pitch_stats = json.load(pitch_f)
 
+    # Add pretrained speaker embeddings (e.g. speaker-verification embeddings)
+    # The embedding order should be the same as manifest order
+    def add_speaker_embedding(self, **kwargs):
+        embedding_path = kwargs.pop("speaker_embedding_path", None)
+        assert embedding_path is not None, "Speaker embedding path is required but got None."
+        self.speaker_embeddings = torch.from_numpy(np.load(embedding_path))
```

Comment:
It sounds like you have a single file with all embeddings in it. Should we not follow the same convention with other sup_data and have each embedding in a separate .pt file?